Library/Homebrew/extend/os/linux/development_tools.rb

```diff
@@ -20,6 +21,7 @@ def default_compiler
       :gcc
     end
 
+    sig { returns(T::Hash[String, T.untyped]) }
```

Comment:
```suggestion
    sig { returns(T::Hash[String, T.nilable(String)]) }
```

and probably append `.to_s.presence` to Glibc version and `.to_s` to oldest CPU.